bokeh/core/compat/bokeh_renderer.py

```diff
@@ -147,7 +147,10 @@ def open_axes(self, ax, props):
 
     def close_axes(self, ax):
         "Complete the axes adding axes-dependent plot props"
-        background_fill_color = convert_color(ax.get_axis_bgcolor())
+        if LooseVersion(mpl.__version__) <= '1.5.9':
```

Comment:
I'm guessing that's when the deprecation warning starts, not when the breaking change happened (just a guess). If the intent is specifically to always avoid the warning, it might very well need to check on a point release.